build/common.legacy.props

```diff
@@ -6,10 +6,13 @@
   <ItemGroup Condition="'$(Shipping)' == 'true' AND '$(IsXPlat)' != 'true'">
     <PackageReference Include="PdbGit" Version="3.0.41" PrivateAssets="all"/>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(DotNetBuildFromSource)' == 'true'">
```

Comment:
Does this work really work with SDK based projects?

Reply:
I removed this and my build is still working, so it doesn't appear to be needed anyway.  I got confused with the net40 issue.